core/arch/decode.h

```diff
@@ -264,6 +264,7 @@ enum {
     OPSZ_32,              /**< 32 bytes: pusha/popa
                            * Also Intel 'qq','pd','ps','x': 32 bytes (256 bits) */
     OPSZ_40,              /**< 40 bytes: 64-bit iret */
+    OPSZ_64,              /**< 64 bytes (512 bits) */
```

Comment:
We can't insert in the middle: that breaks compatibility.  Please use the existing enum location.

Reply:
Really? But do we guarantee the enum's integer value?

Reply:
Yes.  It's an enum in an exposed header.  We can't change the value w/o breaking binary compatibility.

Reply:
Done.